crml/sylo/src/e2ee.rs

```diff
@@ -58,6 +76,12 @@ decl_module! {
 			<PreKeyBundles<T>>::mutate((sender, device_id), |current_pkbs| current_pkbs.extend(pkbs));
 		}
 
+		/// Retrieve and remove the Prekey bundles of a given list of user accounts and devices
+		///
+		/// weight:
+		/// O(n * k) where n is the size of input ```wanted_pkbs```, and k is the number existing PKBS in the storage
```

Comment:
\`wanted_pkbs\`

Reply:
\`\`\`\<comment\>\`\`\` is this triple \` an IDE config thing?

Reply:
yep fixed